ai_interval: 10
resize: 0.2
cb_percentage: 0.8
scale_percent: 0.4